skimage/measure/_marching_cubes.py

```diff
@@ -130,9 +142,6 @@ def marching_cubes(volume, level, spacing=(1., 1., 1.),
     verts = np.asarray(verts)
     faces = np.asarray(faces)
 
-    # Calculate gradient of `volume`, then interpolate to vertices in `verts`
-    grad_x, grad_y, grad_z = np.gradient(volume)
-
```

Comment:
> These days my tools would catch it...

Orthogonal discussion, but I was hinting at whether skimage should use pyflakes in CI. Or is this not done because most code is pyflakes-unreachable Cython anyway?